src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -132,6 +132,7 @@ trait mappedPropVals {
         f.get(this) match {
           case x: String => f.getName -> x
           case y: Option[_] => f.getName -> y.asInstanceOf[Option[_]].getOrElse("").toString
+          case z => f.getName -> z.toString
```

Comment:
it has to go below the Option[_], but you're right it could go above String and eliminate String. I'd like to refactor this later, though, so I'd rather leave it ugly for now.